src/environments/array.js

```diff
@@ -202,8 +201,9 @@ const htmlBuilder = function(group, options) {
             }
 
             if (colDescr.separator === "|") {
-                const separator = stretchy.ruleSpan("vertical-separator", 0.05,
-                    options);
+                const separator = buildCommon.makeSpan(
+                    ["vertical-separator"], [], options
+            );
```

Comment:
The indentation of this line seems to be wrong.

Reply:
Indentation fixed. Thanks for the note.